ModuleFile

```diff
@@ -1,7 +1,8 @@
 name 'citops-repose'
-version '1.8.2'
+version '2.0.0-1'
 description "Repose is an api middleware that provides authentication,
 filtering, ratelimitting and several other features, this deploys it."
 project_page 'https://github.com/rackerlabs/puppet-repose'
 
 dependency 'puppetlabs/stdlib', '>= 2.0.0'
+dependency 'justin8/systemd', '>= 0.4.0'
```

Comment:
@gregswift @mwhahaha per recommendation: switching to camptocamp module. Provides the same functionality I am looking for, so I won't beholden to the module I chose previously.